src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateActionPlanReviewTest.kt

```diff
@@ -261,4 +262,62 @@ class CreateActionPlanReviewTest : IntegrationTestBase() {
         .containsEntry("reference", reviews.completedReviews.first().reference.toString())
     }
   }
+
+  @Test
+  fun `should create a pre release review`() {
+    // Given
+    val earliestCreationTime = OffsetDateTime.now()
+    val prisonerNumber = randomValidPrisonNumber()
+
+    val prisonerFromApi = aValidPrisoner(
+      prisonerNumber = prisonerNumber,
+      legalStatus = LegalStatus.SENTENCED,
+      releaseDate = LocalDate.now().plusMonths(1),
+    )
+    wiremockService.stubGetPrisonerFromPrisonerSearchApi(prisonNumber, prisonerFromApi)
+
+    val reviewSchedule = createReviewScheduleRecord(prisonNumber)
+
+    // When
+    val response = webTestClient.post()
+      .uri(URI_TEMPLATE, prisonNumber)
+      .withBody(
+        aValidCreateActionPlanReviewRequest(
+          prisonId = "MDI",
+          conductedBy = "Barnie Jones",
+          conductedByRole = "Peer mentor",
+          note = "A great review today; prisoner is making good progress towards his goals",
+        ),
+      )
+      .bearerToken(aValidTokenWithAuthority(REVIEWS_RW, username = "auser_gen", privateKey = keyPair.private))
+      .contentType(APPLICATION_JSON)
+      .exchange()
+      .expectStatus()
+      .isCreated
+      .returnResult(CreateActionPlanReviewResponse::class.java)
+
+    // Then
+    val reviews = getActionPlanReviews(prisonNumber)
+    assertThat(reviews)
+      .hasNumberOfCompletedReviews(1)
+      .completedReview(1) {
+        it.wasPreRelease()
```

Comment:
This is the only thing I actually wanted to test - does it hurt to test all the other review fields?